packages/support/src/Colors/ColorManager.php

```diff
@@ -11,6 +11,21 @@ class ColorManager
      */
     protected array $colors = [];
 
+    /**
+     * @var array<int>
+     */
+    protected array $overridingShades = [];
```

Comment:
Maybe use `overridden` instead of `overriding` everywhere

Reply:
`overridden` would be confusing as it signifies the shades that will be overridden instead of the shades that override. Thoughts?

Reply:
Ok ok fair